app/apps/server/communication/rest.js

```diff
@@ -310,7 +310,16 @@ export class AppsRestApi {
 						});
 					} catch (e) {
 						orchestrator.getRocketChatLogger().error('Error getting the App information from the Marketplace:', e.response.data);
-						return API.v1.internalError();
+
+						if (e.response.statusCode >= 500 && e.response.statusCode <= 599) {
```

Comment:
what if we use a function to handle these statuses? as far I can see we are using the same code both places

Reply:
so much better thanks @tassoevan